main/imgSrcs/asteroids/pack.json

```diff
@@ -2,5 +2,5 @@
   minWidth: 512,
```

Comment:
I would leave the JSON configs alone, too. They aren't _actual_ JSON anyways and use a custom parser that might be broken by the lack of trailing commas.